src/rpc/mempool.cpp

```diff
@@ -669,6 +670,133 @@ static RPCHelpMan savemempool()
     };
 }
 
+static RPCHelpMan submitrawpackage()
+{
+    return RPCHelpMan{"submitrawpackage",
+        "\nSubmit a package of raw transactions (serialized, hex-encoded) to local node (-regtest only). Experimental, API may be unstable.\n"
+        "\nThe package will be validated and accepted to mempool if it passes consensus and mempool policy rules. Throws a JSONRPCError if anything goes wrong.\n"
+        "\nWarning: successful submission does not mean the transaction will propagate to other nodes on the network.\n",
+        {
+            {"package", RPCArg::Type::ARR, RPCArg::Optional::NO, "An array of raw transactions.\n",
+                {
+                    {"rawtx", RPCArg::Type::STR_HEX, RPCArg::Optional::OMITTED, ""},
+                },
+            },
+        },
+        RPCResult{
+            RPCResult::Type::OBJ, "", "",
+            {
+                {RPCResult::Type::OBJ_DYN, "tx-results", "transaction results keyed by wtxid",
+                {
+                    {RPCResult::Type::OBJ, "xxxx", "transaction wtxid", {
+                        {RPCResult::Type::STR_HEX, "txid", "The transaction hash in hex"},
+                        {RPCResult::Type::STR_HEX, "wtxid", "The transaction witness hash in hex"},
```

Comment:
If you're already using "wtxid" as a key, maybe not adding it as  "value" ?

Reply:
Because it's possible that the wtxid is different from the key, if there is a same-txid-different-witness tx in the mempool. I could make this more clear maybe?

Reply:
Ok I've renamed it to `other-wtxid` and only use it when communicating that a same-txid-different-witness tx was found. You're right that the wtxid key was redundant otherwise.